pkg/config/default.go

```diff
@@ -168,6 +168,8 @@ const (
 	SeccompOverridePath = _etcDir + "/containers/seccomp.json"
 	// SeccompDefaultPath defines the default seccomp path.
 	SeccompDefaultPath = _installPrefix + "/share/containers/seccomp.json"
+	// DefaultVolumePluginTimeout is the default volume plugin timeout, in milliseconds.
+	DefaultVolumePluginTimeout = 5000
```

Comment:
Why not make this Human?  I don't know Microseconds, but I do know 5s

Reply:
I think people might want to do sub-1s timeouts (250/500 milliseconds seems reasonable) and I didn't want to make this a float. Probably not a big deal to do seconds instead.

Reply:
does 250ms work?

Reply:
Changed to just do seconds... Thinking about it more, it shouldn't really matter that much.